### DISCUSSION TITLE
Not a issue but really need some help. Trying to add pre-text pharses above the character.

### DISCUSSION THREAD

**Discussion by Therealjosephchrzempiec** [networked-aframe/networked-aframe#416] [Q&A] [answered]
Hello, I'm using networked-aframe. It been working great. However What I'm trying to do is post pre messages in like a bubble able the character. Not chat message but a few text phrases. So others can see that. I'm not much of a coder and I really need help. Can someone please help me try to figure this out?

Joseph

**Comment by Therealjosephchrzempiec** [comment]
I was thinking of using the nametag area and temporary replacing the username that is posted to others with a pre phrase text and then after sometime change back to the username. Would that be possible?

**Reply by vincentfretin** [reply, marked as answer]
Hi, you can do that or create another entity above it to put your text.

**Reply by Therealjosephchrzempiec** [reply]
Thank you that did work. I was able to make it all work.

Joseph
